docs/rules/explicit-length-check.md

```diff
@@ -11,11 +11,46 @@ if (!array.length) {}
 ```
 
 
-## Pass
+### Pass
 
 ```js
 if (string.length > 0) {}
 if (array.length > 0) {}
 if (array.length !== 0) {}
 if (array.length === 0) {}
 ```
+
+## Empty Comparisons
+
+Enforce length comparison with `!== 0` when checking for an empty array.
+
+### Fail
+
+```js
+if (string.length < 1) {}
+```
+
+### Pass
+
+```js
+if (array.length !== 0) {}
+```
+
+## Non Zero Comparisons
+
+You can define your preferred way of checking non zero length by providing an option:
+```js
+{
+  "unicorn/explicit-length-check": ["error", {
+    "not-empty": "notEmptyOption"
```

Comment:
`non-zero` seems good 👍 

One thing, since this configuration is done in a `.json` file and since the JSON standard is double-quotes, shouldn't the example also be in double-quotes?